assets/javascript/script.js

```diff
@@ -212,73 +236,79 @@ function drawLoadScreen() {
-        if ((Math.abs(page - currentPage) > 1)) {
-
-            page = Math.floor(page) + (currentPage - page > 1 ? 1 : 0);
-            $('#profile-space').css({ overflow: 'hidden' });
-            setTimeout(function () {
-                $('#profile-space').css({ overflow: '' })
-            }, 10);
-
-            $('#profile-space').scrollLeft(page * window.innerWidth);
-            console.log('scrolling locked to page ' + page)
-        }
-
-        if (!(page % 1) && page != currentPage) { // we have scrolled to a new page
-            console.log(page);
-            console.log(currentPage);
-            $(`#profile-space .profile:nth-child(${currentPage + 1})`).scrollTop(0);
-            currentPage = page;
-        }
-
-        if (page == loadedProfiles.size - 1 && !loading) { // We have just scrolled to the last page
-            loadMore();
-        }
-    });
-}
+  $("#display-area")
+    .empty()
+    .append(
+      $("<div>")
+        .addClass("content")
+        .attr("id", "profile-space")
+    );
 
-$('#splash-button').on('click', function () {
-    $('input[type=file]').trigger('click');
-});
+  $("#profile-space").on("scroll", function(event) {
+    let page = $("#profile-space").scrollLeft() / window.innerWidth;
+    if (Math.abs(page - currentPage) > 1) {
+      page = Math.floor(page) + (currentPage - page > 1 ? 1 : 0);
+      $("#profile-space").css({ overflow: "hidden" });
+      setTimeout(function() {
+        $("#profile-space").css({ overflow: "" });
+      }, 10);
+
+      $("#profile-space").scrollLeft(page * window.innerWidth);
+      console.log("scrolling locked to page " + page);
+    }
 
-$('input[type=file]').change(function (e) {
-    // const vals = $(this).val();
-    // val = vals.length ? vals.split('\\').pop() : '';
-    // const fr = new FileReader();
-    // const bin = fr.readAsBinaryString(val);
-    setupProfileSpace();
-    drawLoadScreen();
-    requestFaceData(e.target.files[0]).then(
-        function (results) {
-            faceData = JSON.parse(results);
-            loadMore();
-        }
-    );
+    if (!(page % 1) && page != currentPage) {
+      // we have scrolled to a new page
+      console.log(page);
+      console.log(currentPage);
+      $(`#profile-space .profile:nth-child(${currentPage + 1})`).scrollTop(0);
+      currentPage = page;
+    }
+
+    if (page == loadedProfiles.size - 1 && !loading) {
+      // We have just scrolled to the last page
+      loadMore();
+    }
+  });
+}
+
+$("#splash-button").on("click", function() {
+  $("input[type=file]").trigger("click");
 });
 
 
+$("input[type=file]").change(function(e) {
+  // const vals = $(this).val();
+  // val = vals.length ? vals.split('\\').pop() : '';
+  // const fr = new FileReader();
+  // const bin = fr.readAsBinaryString(val);
+  setupProfileSpace();
+  drawLoadScreen();
+  requestFaceData(e.target.files[0]).then(function(results) {
+    faceData = JSON.parse(results);
+    loadMore();
+  });
+});
```

Comment:
A lot of the formatting changes make the file less readable overall and create an unnecessarily large diff.